python_interface/design/03_pyHorace_description.md

```diff
@@ -0,0 +1,269 @@
+# A description of the prototype pyHorace implementation
```

Comment:
Overall comments.
1. Please see the comment on MPI at the end and say if it is as critical as I think it may be.
2. While the overall structure is good, I would really like to see a vanilla code example showing the major features going from pyHorace to matlab and back to (e.g. Euphonic) python. The details of why a particular strategy is used obscures clarity in seeing what the final product looks like.
Thanks for doing this -very helpful